src/project/internal/opensaveprojectscenario.cpp

```diff
@@ -103,8 +103,8 @@ RetVal<io::path_t> OpenSaveProjectScenario::askLocalPath(INotationProjectPtr pro
     io::path_t defaultPath = configuration()->defaultSavingFilePath(project, filenameAddition);
 
     std::vector<std::string> filter {
-        trc("project", "MuseScore file") + " (*.mscz)",
-        trc("project", "Uncompressed MuseScore folder (experimental)")
+        trc("project", "MuseScore Studio file") + " (*.mscz)",
+        trc("project", "Uncompressed MuseScore Studio folder (experimental)")
```

Comment:
Didn't we decide not to make this change? (https://mu--se.slack.com/archives/C05HQT2UV2P/p1709112074309509)